bgpd/bgp_nht.c

```diff
@@ -1029,8 +1029,11 @@ static int make_prefix(int afi, struct bgp_path_info *pi, struct prefix *p)
 		break;
 	case AFI_IP6:
 		p->family = AF_INET6;
-
-		if (is_bgp_static) {
+		if (pi && pi->attr && pi->attr->srv6_l3vpn) {
+			IPV6_ADDR_COPY(&(p->u.prefix6),
```

Comment:
no need to check for pi != NULL. it is alread done above.

Reply:
Thanks fixed.